examples/shopping-cart/components/ProductList.vue

```diff
@@ -17,13 +17,20 @@ import { mapGetters, mapActions } from 'vuex'
 
 export default {
   computed: mapGetters({
-    products: 'allProducts'
+    products: 'product/list'
   }),
-  methods: mapActions([
-    'addToCart'
-  ]),
+  methods: {
+    ...mapActions({
+      addItemToCart: 'cart/addItem',
+      decreaseProductInventory: 'product/decreaseInventory'
+    }),
+    addToCart(product) {
+      this.addItemToCart(product);
+      this.decreaseProductInventory(product);
```

Comment:
I think we should create `addToCart` action that commits `ADD_ITEM` and `DECREASE_INVENTORY` mutations, and just dispatch it here. Because it shouldn't be separated that adding an item to the cart and decreasing the inventory of added product.